packages/drawer/src/Default/Default.tsx

```diff
@@ -28,45 +22,34 @@ export const Component = (props: DrawerDefaultProps) => {
             fields: ['theme'],
             value: 'theme',
           },
-          {
-            fields: ['size'],
-            value: 'Size',
-          },
         ],
       },
     },
-    ['prefix', 'theme', 'size']
+    ['prefix', 'theme', 'drawer']
   );
 
-  useEffect(() => {
-    const closeDrawer: HTMLElement | null =
-      document.querySelector('.btn-close');
-
-    if (closeDrawer instanceof Element === false) {
-      return;
-    }
-
-    closeDrawer?.classList.add('owlui-btn-close');
-  });
-
   themePrefixes[baseClass] = `owlui-${baseClass}`;
   themePrefixes['offcanvas'] = 'owlui-offcanvas';
-  themePrefixes['offcanvas-start'] = 'owlui-offcanvas-start';
```

Comment:
Added the `owlui-${baseClass}` prefix in case we want to target something like `owlui-drawer-start` instead of `owlui-offcanvas-start`.